MultithreadingHomework/src/de/telran/Main.java

```diff
@@ -0,0 +1,19 @@
+package de.telran;
+
+public class Main {
+
+    public static void main(String[] args) throws InterruptedException {
+        Counter counter = new Counter();
+
+        ThreadWithThreadClass thread1 = new ThreadWithThreadClass(counter);
+        thread1.start();
+        thread1.join();
```

Comment:
Here you the main() thread is waiting for the thread1 to complete. And only then in line 15 you start the next thread. So, they do not run in parallel, but rather in consequence, so, no issues could be observed. The idea is to say thread1.start()
thread2.start()

and then wait for both
And then check the counter value.